robottelo/cli/ansible.py

```diff
@@ -18,10 +18,22 @@ class Ansible(Base):
 
     @classmethod
     def roles_import(cls, options=None):
-        """Import ansible roles"""
+        """Import ansible roles DEPRECATED"""
```

Comment:
Might be a bit more clear
```suggestion
        """DEPRECATED - Import ansible roles"""
```